site/content/docs/0.0/utilities/shadows.md

```diff
@@ -57,13 +57,19 @@ Shadow semantic tokens as Sass variables.
 
 {{< scss-docs name="ouds-semantic-elevation" file="scss/tokens/_semantic.scss" >}}
 
+#### Composite tokens
```

Comment:
I haven't reviewed everything yet, but just a note here to think to explain what are "composite tokens" in the global main guide.